AmplifyPlugins/Internal/Sources/InternalAWSPinpoint/Endpoint/PinpointClientTypes+Codable.swift

```diff
@@ -0,0 +1,136 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import AWSPinpoint
+import Foundation
+
+extension PinpointClientTypes.EndpointLocation: Codable, Equatable {
```

Comment:
It's a good callout. We don't have any test that explicitly tests this. There might be some that do it indirectly, but I don't know.

I'll add some, should be simple enough since we have a `AmplifyArchiver` utility that we can use.